### DISCUSSION TITLE
Перенос треков в ВК

### DISCUSSION THREAD

**Discussion by Detsle** [Chimildic/YaMuTools#59] [Идея] [answered]
Добрый день, подскажите пожалуйста:
Будет ли реализована поддержка переноса треков в ВК? с ВК в Яндекс.Музыку?
Или например сделать синхронизатор, чтобы и там и там были одни и те же треки?

**Comment by F000x** [comment, marked as answer]
> Добрый день, подскажите пожалуйста: Будет ли реализована поддержка переноса треков в ВК? с ВК в Яндекс.Музыку? Или например сделать синхронизатор, чтобы и там и там были одни и те же треки?

Может, вам будет полезно. Я вот так из вк в Я.Музыку переносил: https://youtu.be/d5RYo1p6t94

**Reply by Detsle** [reply]
А в обратную сторону, как?

**Reply by F000x** [reply]
Обратно не переносил, не знаю

**Reply by Detsle** [reply]
ВК презентовали, красота)
https://vk.cc/music_transfer
